packages/core/src/odata/v2/entity-v2.ts

```diff
@@ -1,18 +1,20 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
 import { EntityBase, Constructable } from '../common';
-import { CustomField } from './selectable/custom-field';
+import { CustomFieldV2 } from './selectable/custom-field-v2';
```

Comment:
```suggestion
import { CustomField } from './selectable/custom-field-v2';
```

Reply:
I am also not sure if this is correct.

Reply:
I think yes, but please correct me. In the `custom-field-v2.ts` file, there is one export: `export { CustomFieldV2 as CustomField };`.
But I guess the decision is to use `V2` as suffix for all classes/interfaces like `ABCV2`. So this comment is not relevant any more.

Reply:
yes this was the idea. It will import the same thing just via two names

Reply:
I remove the suffix if we are in a `v2` `v4` folder.